.travis.yml

```diff
@@ -1,7 +1,10 @@
 language: ruby
 cache: bundler
-sudo: false
+sudo: required
 
+before_install:
```

Comment:
If clamav install ends up slowing Travis down significantly, we may want to exclude it from this process. I think there is a way to still include it in the Gemfile but when given a `TRAVIS` environment variable, clamav would not get built.  See how Scholarsphere does it: https://github.com/psu-stewardship/scholarsphere/blob/f4e46d132c2042d6cad73115773a179f0f56245e/Gemfile#L19